src/components/Reactions/ReportActionItemEmojiReactions.js

```diff
@@ -91,7 +91,8 @@ function ReportActionItemEmojiReactions(props) {
         };
 
         const onReactionListOpen = (event) => {
-            reactionListRef.current.showReactionList(event, popoverReactionListAnchor.current, reactionEmojiName, props.reportActionID);
+            reactionListRef.current.setReactionListReportActionID(props.reportActionID);
+            reactionListRef.current.showReactionList(event, popoverReactionListAnchor.current, reactionEmojiName);
```

Comment:
Why we are making this change?

Reply:
The order didn't change. What I meant is that `setReactionListReportActionID` is async. We should probably `useEffect` to trigger `showReactionList`.

Reply:
To avoid race condition, I think we should

Reply:
@tienifr The latest approach won't work. I forgot that the `useEffect` will only run if the dependency **changes**. If we try to reopen the reaction list for the same report action it won't work.

Reply:
Hmm yeah let's revert for now. I will need to test more. I think even if we `showReactionList` first that's okay because after we call `setReactionListReportActionID` the base reaction list will re-render and in `componentDidUpdate` we `getReactionInformation`.